test/async-await.js

```diff
@@ -249,27 +249,6 @@ function asyncTest (t) {
     })
   })
 
-  test('does not call reply.send() twice if 204 reponse is already sent', t => {
-    t.plan(2)
-
-    const fastify = Fastify()
-
-    fastify.get('/', async (req, reply) => {
-      reply.code(204).send()
-      reply.send = () => {
-        throw new Error('reply.send() was called twice')
-      }
-    })
-
-    fastify.inject({
-      method: 'GET',
-      url: '/'
-    }, (err, res) => {
-      t.error(err)
-      t.equal(res.statusCode, 204)
-    })
-  })
```

Comment:
Can you reverse the test?

Reply:
Why? The test is failing because send is always called.

Reply:
I restore the test and add check for `!reply.send` otherwise `send` is called in each async handler without a return statement 😕